infrastructure/validators/operational-tasks/chill-your-node.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Chill Your Node
+description: Instructions on chilling as a network participant.
+---
+
+Staking bonds can be in any of the three states: validating, nominating, or chilled (neither
+validating nor nominating). When a staker wants to temporarily pause their active engagement in
+staking but does not want to unbond their funds, they can choose to "chill" their involvement and
+keep their funds bonded.
+
+An account can step back from participating in active staking by clicking "Stop" under the Network >
+Staking > Account actions page in [PolkadotJS Apps](https://polkadot.js.org/apps) or by calling the
+[`chill`](https://paritytech.github.io/polkadot-sdk/master/pallet_staking/enum.Call.html#variant.chill) extrinsic in the staking pallet. When an account chooses to chill, it becomes inactive in the next era. The call must be signed by
+the _staking proxy_ account, not the _stash_.
+
+!!!note Primer on stash and staking proxy accounts
+    If you need a refresher on the different responsibilities of the stash and staking proxy account
+    when staking, take a look at the accounts section in the general staking guide.
+
+![staking](/images/infrastructure/operational-tasks/staking-keys-stash-proxy.webp)
+
+## Consideration for Staking Election
+
+A bond that is actively participating in staking but chilled would continue to participate in
+staking for the rest of the current era. If the bond was chilled in sessions 1 through 4 and
+continues to be chilled for the rest of the era, it would NOT be selected for election in the next
+era. If a bond was chilled for the entire session 5, it would not be considered in the next
+election. If the bond was chilled in session 6, its participation in the next era's election would
+depend on its state in session 5.
+
+## Chilling as a Nominator
+
+When you chill after being a nominator, your nominations will be reset. This means that when you
+decide to start nominating again you will need to select validators to nominate once again. These
+can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations
+will not persist across chills.
```

Comment:
```suggestion
won't persist across chills.
```
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.